modules/rs_grading.py

```diff
@@ -433,28 +442,24 @@ def send_lti_grade(assignment, student, lti_record, db):
         # send it back to the LMS
         # have to send a percentage of the max score, rather than total points
         pct = grade.score / float(points) if points else 0.0
-        # print "score", score, points, pct
+        # print("score", score, points, pct)
```

Comment:
Your adding the parenthesis makes me wonder if you are running web2py and the unit tests with python 3???

Reply:
I was confused by the same thing, and I can't really explain this. I printed `sys.version` just before one of these print statements and it's Python 2.7.13. I can run `python rs_grading.py` with no errors, but perhaps web2py's import machinery does something strange?

Reply:
I wonder if there isn't a from __future__ import print_function somewhere in the mix then.  That is a good step in the direction of 3.

Reply:
That's what I'd expect.